### PR TITLE
[EPMLABSBRN-129] - fix audio_file_url setting in load job

### PR DESCRIPTION
**Pull request by RiveriRiv**
[EPMLABSBRN-129](https://jira.epam.com/jira/browse/EPMLABSBRN-129)

- Fixes audio_file_url setting in load job

### CODE REVIEW

File: src/main/kotlin/com/epam/brn/service/parsers/csv/converter/impl/TaskCsvToTaskModelConverter.kt [hunk 1]
```diff
@@ -55,24 +54,25 @@ class TaskCsvToTaskModelConverter : Converter<TaskCsv, Task> {
...
 
         return resources[0]
     }
 
-    private fun createResource(word: String, fileName: String): Resource {
+    private fun createResource(word: String): Resource {
         val resource = Resource()
         resource.word = word
-        resource.audioFileUrl = fileName
+        resource.audioFileUrl = "$word.mp3"
```
Comment (goopnigoop) [inline src/main/kotlin/com/epam/brn/service/parsers/csv/converter/impl/TaskCsvToTaskModelConverter.kt]: please put it to constants or maybe in properties but not here

Comment (ElenaSpb) [reply src/main/kotlin/com/epam/brn/service/parsers/csv/converter/impl/TaskCsvToTaskModelConverter.kt]: as $word is dynamic part (where word is the name of variable)
so logically move only const only ".mp3" part

**Comment by ElenaSpb** [comment]
exception is linked to the task

File: src/main/kotlin/com/epam/brn/service/parsers/csv/converter/impl/TaskCsvToTaskModelConverter.kt (hunk 1 above)
Comment (ElenaSpb) [reply src/main/kotlin/com/epam/brn/service/parsers/csv/converter/impl/TaskCsvToTaskModelConverter.kt]: this need to be set after check that there are no already such resource in db

Comment (ElenaSpb) [reply src/main/kotlin/com/epam/brn/service/parsers/csv/converter/impl/TaskCsvToTaskModelConverter.kt]: so we need to create resource for word in answer array if there are no already resource with such word in db.
we need to create resource for correct word with its filename which is set in file.

**Pull request merged**